Tests/CommonTests/DiagnosticAssemblyScannerTests.cs

```diff
@@ -22,71 +23,120 @@ public class DiagnosticAssemblyScannerTests
         public TestContext TestContext { get; set; }
```

Comment:
This diff isn't very useful - it would be more useful to look at the raw file, as every test has essentially been replaced.